demo/multi_modality_demo.py

```diff
@@ -1,78 +1,101 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import logging
+import os
 from argparse import ArgumentParser
 
-import mmcv
+from mmengine.logging import print_log
 
-from mmdet3d.apis import inference_multi_modality_detector, init_model
-from mmdet3d.registry import VISUALIZERS
+from mmdet3d.apis import MultiModalityDet3DInferencer
 
 
 def parse_args():
     parser = ArgumentParser()
     parser.add_argument('pcd', help='Point cloud file')
-    parser.add_argument('img', help='image file')
-    parser.add_argument('ann', help='ann file')
-    parser.add_argument('config', help='Config file')
-    parser.add_argument('checkpoint', help='Checkpoint file')
+    parser.add_argument('img', help='Image file')
+    parser.add_argument('infos', help='Infos file with annotations')
+    parser.add_argument('model', help='Config file')
+    parser.add_argument('weights', help='Checkpoint file')
     parser.add_argument(
         '--device', default='cuda:0', help='Device used for inference')
     parser.add_argument(
         '--cam-type',
         type=str,
-        default='CAM_FRONT',
+        default='CAM_BACK',
         help='choose camera type to inference')
     parser.add_argument(
-        '--score-thr', type=float, default=0.0, help='bbox score threshold')
+        '--pred-score-thr',
+        type=float,
+        default=0.3,
+        help='bbox score threshold')
     parser.add_argument(
-        '--out-dir', type=str, default='demo', help='dir to save results')
+        '--out-dir',
+        type=str,
+        default='outputs',
+        help='Output directory of prediction results.')
     parser.add_argument(
         '--show',
         action='store_true',
-        help='show online visualization results')
+        help='Show online visualization results')
+    parser.add_argument(
+        '--wait_time',
+        type=float,
+        default=-1,
+        help='The interval of show (s). Demo will be blocked in showing'
+        'results, if wait_time is -1. Defaults to -1.')
+    parser.add_argument(
+        '--no-save-vis',
+        action='store_true',
+        help='Do not save detection vis results')
```

Comment:
What is the difference between detection vis and json results？ Maybe we need to clarify it.

Reply:
visualization result is `.png` and prediction result is `.json` file